plugins/woocommerce/composer.json

```diff
@@ -109,9 +109,11 @@
 	},
 	"scripts": {
 		"post-install-cmd": [
+			"sh ./bin/optimized-email-editor-package.sh",
 			"sh ./bin/post-install.sh"
 		],
 		"post-update-cmd": [
+			"sh ./bin/optimized-email-editor-package.sh",
```

Comment:
This looks like build related, so I suggest to add new core build task named `build:project:email-editor-optimize` and use and try using `rimraf` there to drop unnecessary files.

Reply:
For rimraf, the template inspiration: https://toools.cloud/miscellaneous/glob-tester/ `!/{README.md,src/**/*}`